govc/datastore/rm.go

```diff
@@ -43,6 +43,7 @@ func (cmd *rm) Register(ctx context.Context, f *flag.FlagSet) {
 	cmd.DatastoreFlag, ctx = flags.NewDatastoreFlag(ctx)
 	cmd.DatastoreFlag.Register(ctx, f)
 
+	f.BoolVar(&cmd.kind, "t", true, "Use file type to choose disk or file manager")
```

Comment:
might find out further down, but not sure what will be non-functional if this is set to false.